ReactSkia/sdk/OnScreenKeyBoard.h

```diff
@@ -49,6 +49,8 @@ struct OSKConfig {
   OSKThemes       theme;
   OSKReturnKeyType    returnKeyLabel;
   std::string     placeHolderName;
+  bool          showCursor;
+  std::string   displayString{};
```

Comment:
Can we rename displayString to placeHolderValue, to be relation to member placeHolderName.